README.rst

```diff
@@ -161,6 +158,23 @@ You can also install a pitop subpackage directly in case you don't need the whol
 
   pip3 install pitop.pma
 
+-----------
+Development
+-----------
+
+
+To install the SDK in development mode, the subpackages need to be handled in a particular order, given
+the dependencies between them. Also, the :code:`--extra-index-url` parameter is added to install the :code:`onnxruntime` package
+instead of building it from scratch for some architectures. You can use the :code:`dev-install.sh` script to install the SDK
+in development mode.
```

Comment:
I think it would be helpful to put the command to run at the beginning, the details can come after and are mostly relevent to people where the script does not work for their usecase.

```suggestion
To install the SDK in development mode run the script `dev-install.sh` in the root of the repo:

.. code-block:: bash
  ./dev-install.sh

The script installs the subpackages in a particular order, given
the dependencies between them. Also, the :code:`--extra-index-url` parameter is added to install the :code:`onnxruntime` package
instead of building it from scratch for some architectures.
```